### ISSUE TITLE
Inline suppressions

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#303]
On some occasions suppressing just one reported oelint-adv issue would be a nice feature to have

Proposal

```
# nooelint: <id>[,<id>,...]
some-issue
```
should not report issues for the given ID(s)
